debian/snapcastc.snapcastc-server.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=SnapCastC Server
+After=network.target auditd.service
+
+[Service]
+EnvironmentFile=-/etc/default/snapcastc
+ExecStart=/usr/bin/snapcast-server $SNAPCAST_SERVEROPTS
+ExecReload=/bin/kill -HUP $MAINPID
+KillMode=process
+Restart=on-failure
+RestartPreventExitStatus=255
+Type=notify
```

Comment:
please change to simple

Reply:
just removed, simple is the default of systemd